### ISSUE TITLE
Add project GUID to site deployment analytics

### ISSUE THREAD

**Issue by davidebbo** [projectkudu/kudu#2064]
With the new target files, after deployment, web.config ends with something like this:

```
</configuration>
<!--ProjectGuid: D3DA9554-B9F0-4A3B-A2A0-A0FED0F57619-->
```

We need to take this GUID and add it to our Kudu deployment analytic as a new field.

**Comment by davidebbo** [comment]
@vijayrkn will provide the code that correctly parses the guid out of web.config. There are apparently several variants to the syntax, and we need to make sure they're all covered.

**Comment by shrimpy** [comment]
Just FYI:

Once this change is deployed, we will need to wait for next Azure Deployment in order to see the log consistently. (if a host machine is restarted, we will see logs from this change)


**TL;DR version:**

Kudu is doing dynamic registration for ETW, which will require host machine to restart to clear schema cache. Otherwise new column from this change will not show up unless event "65501" never been invoked before on the machine.
